src/esp/bindings/SimBindings.cpp

```diff
@@ -83,6 +83,9 @@ void initSimBindings(py::module& m) {
       .def_property("frustum_culling", &Simulator::isFrustumCullingEnabled,
                     &Simulator::setFrustumCullingEnabled,
                     R"(Enable or disable the frustum culling)")
+      .def_property("active_dataset", &Simulator::getActiveDatasetName,
+                    &Simulator::setActiveDatasetName,
+                    R"(The currently active dataset being used.)")
```

Comment:
Maybe add a note that setting this will trigger a load of all linked configuration files if a new dataset is requested? Already specified in C++ docs, but again in python API would probably be helpful.